include/uv-win.h

```diff
@@ -483,7 +483,6 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
   union {                                                                     \
     struct {                                                                  \
       /* Used for readable TTY handles */                                     \
-      HANDLE read_line_handle;                                                \
```

Comment:
I think this will break the ABI because the size of `struct rd` is changed. Add a comment of it not being used anymore and we'll remove it for v2.